### PR TITLE
Add wiki paragraphs as input and filter out numbers before training

### PR DESCRIPTION
**Pull request by natsheh**

### CODE REVIEW

**Comment by natsheh** [comment]
@lmartinet  Could you please review and merge if all is fine?

File: IstexDoc2Vec.py [hunk 1]
```diff
@@ -26,9 +26,10 @@ class LoadFileJson() :
 	def __init__(self):
 		self.index = dict()
 		self.count = 0
+		self.wiki_count = 0
 
 # loads randomly and as uniformly as possible in years, n istex doc
-	def LoadDocumentsIstexAndUCBL(self, directory, ucbl) :
+	def LoadDocumentsIstexAndUCBL(self, directory, ucbl, wiki) :
```
Comment (lmartinet) [inline IstexDoc2Vec.py]: It would be better to make the wiki parameter as an option.

File: IstexDoc2Vec.py [hunk 2]
```diff
@@ -72,7 +73,33 @@ def LoadDocumentsIstexAndUCBL(self, directory, ucbl) :
 					self.index["ISTEX"+doc["doi"]] = str(self.count)
 					self.count +=1
 				except : pass
-		print self.count
+## Wikipedia data itteration
+		for sub in os.listdir(wiki):
+			subdir = wiki + sub
+			for fname in os.listdir(subdir):
+				if self.wiki_count > 12000000:
```
Comment (lmartinet) [inline IstexDoc2Vec.py]: The maximum number of paragraphs should be a parameter of the function.

File: IstexDoc2Vec.py [hunk 3]
```diff
@@ -72,7 +73,33 @@ def LoadDocumentsIstexAndUCBL(self, directory, ucbl) :
...
+					break
+				for line in open(os.path.join(subdir, fname)):
+					if len(line.split()) < 2 or line[:8] == '<doc id=':
+						continue
+					else:
+						try:
```
Comment (lmartinet) [inline IstexDoc2Vec.py]: From line 86 to 98, I would prefer to make this part of the code as a separated function.

File: IstexDoc2Vec.py [hunk 4]
```diff
@@ -72,7 +73,33 @@ def LoadDocumentsIstexAndUCBL(self, directory, ucbl) :
...
 					self.count +=1
 				except : pass
-		print self.count
+## Wikipedia data itteration
+		for sub in os.listdir(wiki):
+			subdir = wiki + sub
+			for fname in os.listdir(subdir):
+				if self.wiki_count > 12000000:
+					break
+				for line in open(os.path.join(subdir, fname)):
```
Comment (lmartinet) [inline IstexDoc2Vec.py]: Add a comment to specify that a line corresponds to a paragraph.

Comment (lmartinet) [inline IstexDoc2Vec.py]: For now, better to take the first paragraph for each document (usually corresponding to the introduction with most of the interesting key words).

**Comment by natsheh** [comment]
@lmartinet I have modified the PR according to you comments. Please review. Thanks!

**Pull request merged**